.github/ISSUE_TEMPLATE/funding_application.yml

```diff
@@ -13,55 +13,8 @@ body:
-        7. TAC brings approved proposals to the funding workflow (see below).
-        8. The OpenSSF Staff will review any proposals with TAC technical merit (see Finance evaluation below) and determine which proposals will be funded based on the availability of funds. OpenSSF staff will distribute funds. 
-        9. TAC/Staff will review funding initiative status on milestones. If a funding initiative has more than one milestone, funds may be dispersed after each milestone is achieved (funding initiatives asking for more funds should most likely have more milestones for funding). If the milestone is reached, the TAC/Staff agree that Staff should disperse funds for additional milestones(s).
-  - type: markdown
-    attributes:
-      value: |
-        **TAC review:**
-        - Is this request in line with the [TAC Technical Vision](https://github.com/ossf/tac/blob/main/technical-vision.md)?
-        - Is this request in line with the OpenSSF [MVS](https://openssf.org/about/), and soon to be [finalized R](https://docs.google.com/document/d/1UoQudHQuaXNzakhOYbAS3IceI9TkSSR6N0bgm1fTTK0/edit#heading=h.493lq0mo4y4f)? (this is how GB/GC communicate “top-down” initiatives, like the SOSS Task Force).
-        - Does the requester(s) have an accepted [TI lifecycle phase](https://github.com/ossf/tac/tree/main/process) in good standing?
-          - Within general funding limits for the TIs at the respective lifecycle (e.g. Sandbox should generally ask for less than Incubating, which should ask for less than Graduated)
-          - Within approved categories of asks (e.g. no asking for a TI outing at a fancy restaurant but yes to cloud credits)
-        - What will be the milestone review requested and when?
-        - Final decision is open discussion and a majority consensus vote. (Alternatively, could have a private form vote)
-  - type: markdown
-    attributes:
-      value: |
-        **Finance and Legal Review:**
-        - Who will approve this request?
-          - GM up to $100k, B&F committee up to $250k, GB above $250k. These amounts are for the entirety of the funding request.
-        - Is funding available in the this category?
-          - If yes, recommend to support?
-            - How much funds will be available for other requests? (Should we approve one request for all the funds, or seek to approve multiple smaller requests)?
-          - If no, recommend to route through an alternate source?
-  - type: markdown
-    attributes:
-      value: |
-        **Communication of Final Decision:**
-        - Complete like a CFP. Notify in writing if it was accepted or not accepted. Put out a list of accepted proposals (like the events schedule).
+        **Process:**
+        - Please review the process in the [TAC Process Documents](https://github.com/ossf/tac/blob/main/process/TI%20Funding%20Requst%20Process.md)
```

Comment:
if/when https://github.com/ossf/tac/pull/302 merges, this link will need to be fixed (since the filename will change)

Reply:
Good call, let's keep this open until that closes.